.github/workflows/first-interaction.yml

```diff
@@ -0,0 +1,17 @@
+name: "First Time Contributor"
+on:
+  schedule:
+  - cron: "*/5 * * * *"
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: fjeremic/cron-first-interaction@master
```

Comment:
Is this personal reference correct?

Reply:
Actually we should really use a fixed release. I'll make that change. The personal repository is the correct reference. This is a publicly available GitHub Action, created for use by any open sourced project. It overcomes the limitations of the official actions/first-interaction Action. You can read more about the limitation it overcomes in the README [1].

[1] https://github.com/fjeremic/cron-first-interaction#cron-first-interaction

Reply:
Fixed in [force-push](https://github.com/eclipse/omr/compare/02fed63e6206221d73bd82917a10bc0ac274c3f6..94f6fb6ea30c69ca960f027c5b75e14f2cf63186). We are now using fixed releases.